### PR TITLE
Replace test.sh with shell command `rspec spec`

### PR DESCRIPTION
**Pull request by myacheg**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,5 +2,5 @@ language: ruby
 rvm:
   - 2.3.4
 script:
-  - sh test.sh
+  - bundle exec rspec spec
 cache: bundler
```
Comment (lesykgg) [inline .travis.yml]: newline at the end?

File: Gemfile [hunk 2]
```diff
@@ -6,3 +6,7 @@ gem 'activemodel', '4.2.7'
 group :development do
  gem 'rake'
 end
+
+group :test do
+  gem 'rspec'
+end
```
Comment (lesykgg) [inline Gemfile]: same here

**Review by lesykgg — APPROVED** [review]

**Review by vbyno — APPROVED** [review]

File: Gemfile (hunk 2 above)
Comment (myacheg) [reply Gemfile]: done

File: .travis.yml (hunk 1 above)
Comment (myacheg) [reply .travis.yml]: done

**Review by dkh — APPROVED** [review]

**Pull request merged**